booking-ui/src/styles/Search.css

```diff
@@ -163,6 +163,10 @@
     background-color: rgba(184, 37, 222, 0.9) !important;
 }
 
+.space-buddy-desk {
+    background-color: rgba(36, 21, 197, 0.9) !important;
```

Comment:
I chose this color but not really familiar with accessibility impact, happy to change with another shade / color.